README.md

```diff
@@ -1,8 +1,10 @@
 ![](https://img.shields.io/badge/Microverse-blueviolet)
 
-# Mobile_Project
+# Desktop_version
```

Comment:
[OPTIONAL] It's better to use a name that described your whole project e.g. `My Portfolio` or `Portfolio` 🤗